SU2_CFD/src/output/COutput.cpp

```diff
@@ -0,0 +1,1980 @@
+    
+    return false;
+    
+  }
+  
+  /*--- Check if screen output should be written --- */
+  
+  if (!PrintOutput(curTimeIter, ScreenWrt_Freq_Time)&& 
+      !(curTimeIter == config->GetnTime_Iter() - 1)){
+    
+    return false;
+    
+  }
+  
+  if (convergence) {return true;}
+  
+  if (!PrintOutput(curOuterIter, ScreenWrt_Freq_Outer) && 
+      !(curOuterIter == config->GetnOuter_Iter() - 1)){
+    
+    return false;
+    
+  }
+  
+  if (!PrintOutput(curInnerIter, ScreenWrt_Freq_Inner) &&
+      !(curInnerIter == config->GetnInner_Iter() - 1)){
+    
+    return false;
+    
+  }
+ 
+  return true;
+  
+}
+
+bool COutput::WriteHistoryFile_Output(CConfig *config) { 
+
+  unsigned long HistoryWrt_Freq_Inner = config->GetHistory_Wrt_Freq(2);
+  unsigned long HistoryWrt_Freq_Outer = config->GetHistory_Wrt_Freq(1);
+  unsigned long HistoryWrt_Freq_Time  = config->GetHistory_Wrt_Freq(0);    
+    
+  /*--- Check if screen output should be written --- */
+  
+  if (!PrintOutput(curTimeIter, HistoryWrt_Freq_Time)&& 
+      !(curTimeIter == config->GetnTime_Iter() - 1)){
+    
+    return false;
+    
+  }
+  
+  if (convergence) {return true;}
+  
+  if (!PrintOutput(curOuterIter,HistoryWrt_Freq_Outer) && 
+      !(curOuterIter == config->GetnOuter_Iter() - 1)){
+    
+    return false;
+    
+  }
+  
+  if (!PrintOutput(curInnerIter, HistoryWrt_Freq_Inner) &&
+      !(curInnerIter == config->GetnInner_Iter() - 1)){
+    
+    return false;
+    
+  }
+ 
+  return true;
+
+}
+
+bool COutput::WriteVolume_Output(CConfig *config, unsigned long Iter){
+  if (config->GetTime_Domain()) return ((Iter % config->GetVolume_Wrt_Freq() == 0));
+  else {
+     return ((Iter > 0) && (Iter % config->GetVolume_Wrt_Freq() == 0));
+  }
+}
+
+void COutput::SetCommonHistoryFields(CConfig *config){
+  
+  /// BEGIN_GROUP: ITERATION, DESCRIPTION: Iteration identifier.
+  /// DESCRIPTION: The time iteration index.
+  AddHistoryOutput("TIME_ITER",     "Time_Iter",  ScreenOutputFormat::INTEGER, "ITER", "Time iteration index"); 
+  /// DESCRIPTION: The outer iteration index.
+  AddHistoryOutput("OUTER_ITER",   "Outer_Iter",  ScreenOutputFormat::INTEGER, "ITER", "Outer iteration index"); 
+  /// DESCRIPTION: The inner iteration index.
+  AddHistoryOutput("INNER_ITER",   "Inner_Iter", ScreenOutputFormat::INTEGER,  "ITER", "Inner iteration index"); 
+  /// END_GROUP
+  
+  /// BEGIN_GROUP: TIME_DOMAIN, DESCRIPTION: Time integration information
+  /// Description: The current time
+  AddHistoryOutput("CUR_TIME", "Cur_Time", ScreenOutputFormat::SCIENTIFIC, "TIME_DOMAIN", "Current physical time (s)");
+  /// Description: The current time step
+  AddHistoryOutput("TIME_STEP", "Time_Step", ScreenOutputFormat::SCIENTIFIC, "TIME_DOMAIN", "Current time step (s)");
+ 
+  /// DESCRIPTION: Currently used wall-clock time.
+  AddHistoryOutput("PHYS_TIME",   "Time(sec)", ScreenOutputFormat::SCIENTIFIC, "PHYS_TIME", "Average wall-clock time"); 
```

Comment:
Did 'WALL_TIME' change to 'PHYS_TIME'?

Reply:
Don't know what the best name would be. Maybe WALL_TIME is more appropriate.